src/EventSubscriber/EventSubscriber.php

```diff
@@ -120,6 +120,7 @@ public function onKernelRequest(RequestEvent $event) {
 
     if (
       $event->getRequestType() == HttpKernelInterface::MAIN_REQUEST &&
+      !getenv('LANDO') &&
```

Comment:
Tests don't run locally without something like this to override the redirect.

Reply:
Yeah this was annoying for me too. why don't we use something like `Settings::get('stanford_capture_ownership', FALSE)`. Then we just have to add a line into the settings.php file for prod environments.

Reply:
That's a good idea.  I've made the change.

Reply:
well, that's broken the test for the event subscriber.

Reply:
https://www.drupal.org/docs/automated-testing/phpunit-in-drupal/unit-testing-more-complicated-drupal-classes#s-mocking-values-in-the-settingsphp-during-tests
in
https://github.com/SU-SWS/stanford_profile/blob/11.x/tests/src/Kernel/EventSubscriber/EventSubscriberTest.php#L138

Reply:
Gotcha.  Thanks for the tip.